appium-app/testng/examples/biometric-login/pom.xml

```diff
@@ -6,7 +6,7 @@
         <artifactId>demo-java</artifactId>
         <groupId>com.saucelabs</groupId>
         <version>1.0-SNAPSHOT</version>
-        <relativePath>../../../../../pom.xml</relativePath>
+        <relativePath>../../../../pom.xml</relativePath>
```

Comment:
@eyaly is the plan to create a pom.xml for every single sauce labs feature? biometric, test status update, performance, rdc ios, rdc android...

Reply:
Thanks @nadvolod for reviewing this PR. 
1. Each module is an independent package that can be used and run without any dependency on the other modules so it is around 8-10 files for a Mobile module ( 1 x readme, 1 x pom, 1 x TestNG XML file android, 1 x TestNG XML file iOS, 1 x test class for android,   1 x test class for iOS, 1 x page class for android,   1 x page class for iOS, 1 x utils file  = 9 files ) 
For VM there is no need for Android and iOS so it will be fewer files. 
2. The plan is to create modules for features that are needed by our customers.  If you search in Slack, you will find out that CSM and support asked a few times for image-injection and biometric-auth. Mostly Wim sent a link to his demo-js/webdriverIO packages ( similar to modules) 
I can think also about W3C,  running in parallel, basic Appium script to run a test on Sauce UP that I want to continue and push to our repo

Reply:
@eyaly I understand the need for examples. We have that in all the repos. What do you think of this https://github.com/saucelabs-training/demo-java/pull/72?

Reply:
Hi @nadvolod . From #72 you want modules such as:  appium-testng-examples ,  appium-junit4-examples, appium-junit5-examples and ALL the examples will be under one modules. 
We can try something like that but instead of appium-testng-examples it will be appium-app-testng-examples for App examples. For testNG I will create a separate TestNG XML file per feature in this module ( so we can tell customers how to run one feature for specific OS under this module).  If this sounds good to you, I will do the needed changes ( a lot of changes... :-| )

Reply:
@eyaly it's doesn't need to be `appium-app-testng-examples` because we can do [this](https://github.com/saucelabs-training/demo-java/tree/master/appium-junit4-examples/src/test/java/com). No need to have 2 different modules for Appium as the only difference is in what you do in your test code. It's:


Appium
- Emusim
  - WebFeature.java
  - NativeAppFeature.java 
- RealDevices
   - NativeAppFeature.java
   - WebAppFeature.java
   - BiometricAuth.java
   - ...

I don't understand "_For testNG I will create a separate TestNG XML file per feature in this module ( so we can tell customers how to run one feature for specific OS under this module). If this sounds good to you, I will do the needed changes_". 

Can you kindly create a small PR (if possible, so you don't do too much work) and show me what you mean?

When I have time, next week, I will take your examples from https://github.com/saucelabs-training/demo-java/pull/71 and put them into https://github.com/saucelabs-training/demo-java/pull/72 so that you see what I mean.

Reply:
Thanks @nadvolod . 
I will try and create the needed changes. 
There is no need for separate folders and files for RDC and EmuSim. 
Mostly it is the same code ( just the capabilities can be different) 

if (rdc) {
            System.out.println("Sauce - Run on real device");
            capabilities.setCapability("deviceName", "iPhone 8*");
            // Enable touchID
            capabilities.setCapability("allowTouchIdEnroll", true);

            //      You can use  storage:filename=" +appName if you uploaded your app to Saucd Storage
            //capabilities.setCapability("app", "storage:filename=" +appName);
            capabilities.setCapability("app", "https://github.com/saucelabs/sample-app-mobile/releases/download/2.7.1/iOS.RealDevice.SauceLabs.Mobile.Sample.app.2.7.1.ipa");
        } else { // Simulator
            System.out.println("Sauce - Run on virtual mobile device");
            capabilities.setCapability("deviceName", "iPhone 8 Simulator");
            capabilities.setCapability("platformVersion", "13.4");
            //      You can use  storage:filename=" +appName if you uploaded your app to Saucd Storage
          //  capabilities.setCapability("app", "sauce-storage:" + appName);
            capabilities.setCapability("app", "https://github.com/saucelabs/sample-app-mobile/releases/download/2.7.1/iOS.Simulator.SauceLabs.Mobile.Sample.app.2.7.1.zip");
        }